fdbserver/DataDistribution.actor.h

```diff
@@ -390,19 +390,21 @@ struct StorageWiggleMetrics {
 };
 
 struct StorageWiggler : ReferenceCounted<StorageWiggler> {
+	enum State : uint8_t { INVALID = 0, RUN, PAUSE };
+	AsyncVar<bool> nonEmpty;
 	DDTeamCollection const* teamCollection;
 	StorageWiggleMetrics metrics;
-
 	// data structures
 	typedef std::pair<StorageMetadataType, UID> MetadataUIDP;
 	// min-heap
 	boost::heap::skew_heap<MetadataUIDP, boost::heap::mutable_<true>, boost::heap::compare<std::greater<MetadataUIDP>>>
 	    wiggle_pq;
 	std::unordered_map<UID, decltype(wiggle_pq)::handle_type> pq_handles;
 
-	AsyncVar<bool> nonEmpty;
+	State _state = State::INVALID;
```

Comment:
`_state` is bit awkard from a style perspective, to avoid the reserved word collision how about `status`, which is used in several other components to represent the execution state of a thing.

Reply:
Probably would be best to rename `State` to `Status` as well.